### ISSUE TITLE
Bug: Playground: cannot modify tsconfig or eslintrc anymore

### ISSUE THREAD

**Issue by kkmuffme** [typescript-eslint/typescript-eslint#9452]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Abug) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Relevant Package

website

### Playground Link

https://typescript-eslint.io/play/#ts=5.5.2&fileType=.ts&code=MoUwLgPAdgrgtgIxAJwHwG4BQBGABAQwGddZEUsB3ASygBMB7CgQnSA&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQHYHsBaWXWZRAE0MQA8AHaFZAS311Q0nwDNvIAacNkgABAC4BPOigDG0ZnTHVk8ZrjEB6AsVLkqtBk1bt0URNGj5okIQF8QtoA&tsconfig=&tokens=false

### Repro Code

```TypeScript
https://typescript-eslint.io/play/#ts=5.5.2&fileType=.ts&code=MoUwLgPAdgrgtgIxAJwHwG4BQBGABAQwGddZEUsB3ASygBMB7CgQnSA&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQHYHsBaWXWZRAE0MQA8AHaFZAS311Q0nwDNvIAacNkgABAC4BPOigDG0ZnTHVk8ZrjEB6AsVLkqtBk1bt0URNGj5okIQF8QtoA&tsconfig=&tokens=false

Clicking on "tsconfig" or "eslintrc" does nothing (same for clicking on "Visual editor"), which renders the playground useless.
No console errors

Chrome 126
```

### Expected Result

Modify tsconfig and eslintrc

### Actual Result

Doesn't work

### Versions

Playground

**Comment by kirkwaiblinger** [comment]
@kkmuffme 
Ah, this is basically the same as https://github.com/typescript-eslint/typescript-eslint/issues/9451. Same response - I'm not personally able to repro... I'm on Mac Chrome 126. We'll see if anyone else experiences this 🤔

**Comment by bradzacher** [comment]
Same - works fine for me!

![Screenshot_20240629-100100.png one](https://github.com/typescript-eslint/typescript-eslint/assets/7462525/f48a5449-d5e9-41e1-8579-0ca74b56274e)

![Screenshot_20240629-100100.png two](https://github.com/typescript-eslint/typescript-eslint/assets/7462525/5478ca17-24ed-4476-b061-3c556e694d93)
